docs/topics/migrations.txt

```diff
@@ -758,13 +766,11 @@ If you already have pre-existing migrations created with
 * Run ``python manage.py migrate --fake-initial``. Django will see that the
   tables for the initial migrations already exist and mark them as applied
   without running them. (Django won't check that the table schema match your
-  models, just that the right table names exist).
+  models, just that the right table names and field names exist).
```

Comment:
I think this overstates the capabilities of the detection, because it implies that field names are always checked, which is not true. For a `CreateModel` operation, even in this PR we still don't check that the field names match up if the table exists. We only check field names in the edge case of a separate `AddField` operation in a split initial migration.

I think it's important that we _not_ overstate this, so that people know they need to ensure the schema matches themselves. Rather than going into details, I would just leave the previous wording in place here.